### PR TITLE
lesson_26

### PR DESCRIPTION
**Pull request by test-python-student-0**

### CODE REVIEW

File: student_0/lesson_26_Search and hashing algorithms/temp_tasks.py [hunk 1]
```diff
@@ -0,0 +1 @@
+;dsljkf;asjd;as
```
Comment (podolyanyn) [inline student_0/lesson_26_Search and hashing algorithms/temp_tasks.py:1]: bad code !

**Comment by podolyanyn** [comment]
bad work

File: student_0/lesson_26_Search and hashing algorithms/temp_tasks.py (hunk 1 above)
Comment (StionRa) [reply student_0/lesson_26_Search and hashing algorithms/temp_tasks.py:1]: hi

Comment (stargazinn) [inline student_0/lesson_26_Search and hashing algorithms/temp_tasks.py:1]: good code

**Pull request merged**